### DISCUSSION TITLE
How to use this.$patch in Setup Stores

### DISCUSSION THREAD

**Discussion by max5432112345** [vuejs/pinia#1531] [Help and Questions] [answered]
### Reproduction

---

### Steps to reproduce the bug

In vuex i do years ago

```
    data[key] = {
      count: 0,
      items: [],
    }
    data = Object.assign({}, data) // or vue set, may be now is not required that
```

### Expected behavior

mutate state

### Actual behavior

all uses options api in examples

**Comment by max5432112345** [comment, marked as answer]
var.value

**Reply by serialine** [reply]
https://github.com/vuejs/pinia/discussions/1012#discussioncomment-2084271

**Reply by max5432112345** [reply]
It work without it. At that moment i not use` var.value `
